src/forcemove.cpp

```diff
@@ -0,0 +1,172 @@
+#include "forcemove.h"
+#include "random.h"
+namespace Faunus {
+
+namespace Move {
+
+/**
+ * @brief Compute a single dimension contribution to a mean square thermal speed of a particle, i.e., compute 〈v²_x〉.
+ *
+ * This is equivalent to a one third of a total mean square thermal speed of a particle in three dimensions,
+ * i.e., 〈v²〉 = 3 × 〈v²_x〉.
+ *
+ * @param mass particle's mass in dalton
+ * @return a mean square thermal speed in ångström squared per picosecond squared
+ */
+template<typename T=double> static inline T meanSquareSpeedComponent(T mass) {
+    return (pc::kT() / mass * 1.0_kg) * ((1.0_m * 1.0_m) / (1.0_s * 1.0_s));
+}
+
+/**
+ * @brief Generate standard normal random vector
+ */
+class NormalRandomVector {
```

Comment:
Class declaration should be in header file

Reply:
Moved. Before it was only a “private” class to be used within the translation unit. Looking forward to modules that are introduced in C++20.